.gitlab-ci.yml

```diff
@@ -41,6 +41,18 @@ build:docker:
     - when: never
   script: ["true"]
 
+build:debian:
+  rules:
+    - when: manual
+      allow_failure: true
+  image: "golang:${GO_VERSION}"
```

Comment:
Also, I recommend using a Debian image instead to not get unexpected major OS update with a golang image update.

Reply:
This comment is now obsolete as we moved to tarballs.